### PR TITLE
nifi-registry: depend on openjdk

### PR DESCRIPTION
**Pull request by dtrodrigues**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/nifi-registry.rb [hunk 1]
```diff
@@ -5,19 +5,24 @@ class NifiRegistry < Formula
...
+
   def install
     libexec.install Dir["*"]
     rm Dir[libexec/"bin/*.bat"]
 
     bin.install libexec/"bin/nifi-registry.sh" => "nifi-registry"
-    bin.env_script_all_files libexec/"bin/", NIFI_REGISTRY_HOME: libexec
+    bin.env_script_all_files libexec/"bin/",
+                             NIFI_REGISTRY_HOME: libexec,
+                             JAVA_HOME:          Formula["openjdk"].opt_prefix
```
Comment (SMillerDev) [inline Formula/nifi-registry.rb:25]: I think this should be overwrittable

Comment (dtrodrigues) [reply Formula/nifi-registry.rb:25]: What do you mean by that?

Comment (SMillerDev) [reply Formula/nifi-registry.rb:25]: Something like https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/jboss-forge.rb#L15

**Review by SMillerDev — APPROVED** [review]
Thanks @dtrodrigues  ! Without your contributions it'd be impossible to keep homebrew going. You can feel good knowing that you've made the world a lot better for homebrew users around the world! 👍 🎉

**Pull request merged**